src/components/parameters/ParameterBaseDropdown.svelte

```diff
@@ -53,17 +72,32 @@
 
 <div class="parameter-base-dropdown" style="grid-template-columns: {columns}">
   <ParameterName {formParameter} />
-  <SearchableDropdown
-    bind:selectedOptionValues={selectedDropdownOptions}
-    {allowMultiple}
-    {disabled}
-    showPlaceholderOption={false}
-    options={dropdownOptions}
-    placeholder={disabled ? '' : placeholder}
-    {searchPlaceholder}
-    on:change={onChange}
-  />
+  <!--  TODO: If disabled, use an input to avoid SearchableDropdown need to map the value to existing options, this way it can report accurately the value that it was used. Is this the endgame decision? -->
+  {#if disabled}
+    <Input>
+      <input
+        value={Array.isArray(formParameter.value) ? formParameter.value.join(', ') : formParameter.value}
+        class="st-input w-full"
+        class:error={formParameter.errors !== null}
+        aria-label={formParameter.name}
+        {disabled}
+        type="text"
+      />
+    </Input>
+  {:else}
+    <SearchableDropdown
+      bind:selectedOptionValues={selectedDropdownOptions}
+      {allowMultiple}
+      {disabled}
```

Comment:
Can we remove this `{disabled}` just to avoid confusion? I know it's moot since it won't get here if it is disabled, but it's just for clarity.